middleware/handlePost.js

```diff
@@ -1,8 +1,98 @@
+const axios = require('axios');
 const {postWinners, postLosers} = require('./crud-functions/index.js');
 
 async function handlePost(ctx) {
-  const {channel_id, text} = ctx.request.body;
-  postWinners(text);
+  const {user_id, response_url, text: correctAnswer} = ctx.request.body;
+
+  const adminIds = process.env.ADMIN_IDS.split(',');
+  if (!adminIds.includes(user_id)) {
+    ctx.status = 200;
```

Comment:
Shouldn't this return 401?

Reply:
If we return 401, Slack just displays a "generic" error message. I'd rather return successfully with a message that the Slack user can easily understand. 

<img width="809" alt="Screen Shot 2019-04-07 at 11 47 36 AM" src="https://user-images.githubusercontent.com/29233959/55686081-29b98280-592b-11e9-9c1f-f7481860e008.png">